SofascoreAcademy/SofascoreAcademy/Views/SettingsView/SettingsViewController.swift

```diff
@@ -0,0 +1,113 @@
+import Foundation
+import UIKit
+import SnapKit
+import SofaAcademic
+
+class SettingsViewController: UIViewController {
+    
+    private let blueContainer = UIView()
+    private let settingsHeader = SettingsHeaderView()
+    private let aboutView = SettingsAboutView()
+    
+    private let appLogo = UIImageView(image: UIImage(named: "sofascore_logo_blue"))
+    
+    private let logoutView = UIView()
+    private let logoutText = UILabel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        addViews()
+        styleViews()
+        setupConstraints()
+        setupGestureRecognizers()
+        settingsHeader.eventDelegate = self
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(true, animated: animated)
```

Comment:
zasto ovo sakrivas, mogao si koristiti nav bar i njegov back button?

Reply:
razmislja sam da imam vise moci nad headerom ako je custom nego ako je default od iosa, jel to krivo?